### DISCUSSION TITLE
How to enable PiVPN (Wireguard) to auto start on boot?

### DISCUSSION THREAD

**Discussion by almog-co** [pivpn/pivpn#1352] [Q&A]
I'm running PiVPN on an Ubuntu VM. Sometimes, my server goes down and the Ubuntu VM will successfully startup, but the VPN will remain down until I run systemctl to manually start it. 

Is there a setting to start it on boot before even logging in?

**Comment by coolapso** [comment]
`sudo systemctl enable openvpn`
